server_extensions_api/src/main/java/com/generalbytes/batm/server/extensions/IIdentitySimple.java

```diff
@@ -0,0 +1,155 @@
+/*************************************************************************************
+ * Copyright (C) 2014-2021 GENERAL BYTES s.r.o. All rights reserved.
+ *
+ * This software may be distributed and modified under the terms of the GNU
+ * General Public License version 2 (GPL2) as published by the Free Software
+ * Foundation and appearing in the file GPL2.TXT included in the packaging of
+ * this file. Please note that GPL2 Section 2[b] requires that all works based
+ * on this software must also be made publicly available under the terms of
+ * the GPL2 ("Copyleft").
+ *
+ * Contact information
+ * -------------------
+ *
+ * GENERAL BYTES s.r.o.
+ * Web      :  http://www.generalbytes.com
+ *
+ ************************************************************************************/
+package com.generalbytes.batm.server.extensions;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+public interface IIdentitySimple {
```

Comment:
Please add javadoc explaining purpose of this interface

Also please consider renaming it to `IIdentityBase`, `IIdentityWithoutRelations`, etc.